src/app/dashboard-widgets/add-codebase-widget/add-codebase-widget.component.html

```diff
@@ -20,16 +24,20 @@ <h2 class="card-pf-title">
     <div class="card-pf-body f8-card-body">
       <f8-feature-toggle featureName="AppLauncher">
         <div class="f8-blank-slate-card" *ngIf="codebases.length === 0" default-level>
-            <h3>This Space has no Codebases</h3>
-            <div class="f8-blank-slate-main-action">
-              <button id="spacehome-codebases-create-button" class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add Codebase</button>
+          <h3>This Space has no Codebases</h3>
+          <div class="f8-blank-slate-main-action">
+            <div *ngIf="userOwnsSpace">
+              <button id="default-level-spacehome-codebases-create-button" class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add Codebase</button>
             </div>
+          </div>
         </div>
         <div class="f8-blank-slate-card" *ngIf="codebases.length === 0" user-level>
-            <h3>This Space has no Codebases</h3>
-            <div class="f8-blank-slate-main-action">
-              <button class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add Codebase</button>
+          <h3>This Space has no Codebases</h3>
+          <div class="f8-blank-slate-main-action">
+            <div *ngIf="userOwnsSpace">
+              <button id="user-level-spacehome-codebases-create-button" class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add Codebase</button>
             </div>
+          </div>
         </div>
```

Comment:
Lines 25-41

Again, the only difference here is the button ID. We should clean this up and just remove the AppLauncher feature.

```
<div class="f8-blank-slate-card" *ngIf="codebases.length === 0" default-level>
          <h3>This Space has no Codebases</h3>
          <div class="f8-blank-slate-main-action">
            <div *ngIf="userOwnsSpace">
              <button id="default-level-spacehome-codebases-create-button" class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add Codebase</button>
            </div>
          </div>
        </div>
        <div class="f8-blank-slate-card" *ngIf="codebases.length === 0" user-level>
          <h3>This Space has no Codebases</h3>
          <div class="f8-blank-slate-main-action">
            <div *ngIf="userOwnsSpace">
              <button id="user-level-spacehome-codebases-create-button" class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add Codebase</button>
            </div>
          </div>
</div>
```